Sources/Sentry/SentryHttpTransport.m

```diff
@@ -291,29 +297,40 @@ - (void)deleteEnvelopeAndSendNext:(NSString *)envelopePath
     SENTRY_LOG_DEBUG(@"Deleting envelope and sending next.");
     [self.fileManager removeFileAtPath:envelopePath];
     self.isSending = NO;
+
+    __weak SentryHttpTransport *weakSelf = self;
     [self.dispatchQueue dispatchAfter:cachedEnvelopeSendDelay
-                                block:^{ [self sendAllCachedEnvelopes]; }];
+                                block:^{
+                                    if (weakSelf == nil) {
+                                        return;
+                                    }
```

Comment:
l: This one could be removed since this only would result in a message send to `nil`, with no logs.